easybuild/tools/github.py

```diff
@@ -958,6 +958,53 @@ def not_eligible(msg):
     return res
 
 
+def close_pr(pr, reason):
+    """
+    Close specified pull request
+    """
+    github_user = build_option('github_user')
+    if github_user is None:
+        raise EasyBuildError("GitHub user must be specified to use --close-pr")
+
+    pr_target_account = build_option('pr_target_account')
+    pr_target_repo = build_option('pr_target_repo')
+
+    pr_url = lambda g: g.repos[pr_target_account][pr_target_repo].pulls[pr]
+    status, pr_data = github_api_get_request(pr_url, github_user)
+    if status != HTTP_STATUS_OK:
+        raise EasyBuildError("Failed to get data for PR #%d from %s/%s (status: %d %s)",
+                             pr, pr_target_account, pr_target_repo, status, pr_data)
+    
+    if pr_data['state'] == 'closed':
+        raise EasyBuildError("PR #%d from %s/%s is already closed.", pr, pr_target_account, pr_target_repo)
+
+    msg = "\n%s/%s PR #%s was submitted by %s, " % (pr_target_account, pr_target_repo, pr, pr_data['user']['login'])
+    msg += "you are using GitHub account '%s'\n" % github_user
+    print_msg(msg, prefix=False)
+
+    dry_run = build_option('dry_run') or build_option('extended_dry_run')
+
+    if not reason:
+        reason = "unspecified"
+
+    comment = "@%s, this PR is being closed for the following reason: %s." % (pr_data['user']['login'], reason)
+    comment += " Do reopen if the PR is still relevant."
```

Comment:
`Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.`

Reply:
and a link to the new section in the documentation?

Reply:
Or better `our policy`